packages/apps/spaces/components/ui-kit/organisms/timeline/Timeline.tsx

```diff
@@ -93,14 +96,10 @@ export const Timeline = ({
   };
 
   return (
-    <div className='mt-5 mb-3'>
-      <div className='text-sm text-gray-500 flex justify-content-center mb-1'>
-        {' '}
-        Now{' '}
-      </div>
+    <div className='mb-3'>
       {loggedActivities.map((e: any, index) => (
-        <React.Fragment key={e.id}>
-          {getTimelineItemByTime(e.type, e, index)}
+        <React.Fragment key={uuidv4()}>
```

Comment:
Why this change?

Reply:
conversations don't have an id property. this will be fixed later when we move to a unified endpoint for timeline